Admin/AbstractAdmin.php

```diff
@@ -1490,13 +1509,28 @@ public function getLabel()
 
     /**
      * @param bool $persist
+     *
+     * @deprecated since 3.x, to be removed in 4.0.
      */
     public function setPersistFilters($persist)
     {
+        @trigger_error(
+            'The '.__METHOD__.' method is deprecated since version 3.x and will be removed in 4.0.',
+            E_USER_DEPRECATED
+        );
+
         $this->persistFilters = $persist;
```

Comment:
Can you find a BC way for this? Maybe set the `filterPersister` to null or to a `FilterPersisterInterface` instance